src/datadog/datadog_agent.cpp

```diff
@@ -263,5 +330,35 @@ void DatadogAgent::flush() {
   }
 }
 
+void DatadogAgent::send_app_started(nlohmann::json&& tracer_config) {
+  auto payload = tracer_telemetry_->app_started(std::move(tracer_config));
+  auto post_result = http_client_->post(
+      telemetry_endpoint_, telemetry_set_request_headers_, std::move(payload),
+      telemetry_on_response_, telemetry_on_error_);
+  if (auto* error = post_result.if_error()) {
+    logger_->log_error(*error);
```

Comment:
Reconsider using `error.with_prefix` here and below, as you did above (to distinguish telemetry stuff from trace stuff, and to distinguish among the telemetry stuff).